src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -41,6 +41,19 @@ Correlate any general patterns on the top 5 errors chart to alerts occurring dur
 * **Dynamic grouping:** The default grouping for error occurrences is based on error message, error class, and transaction UI name. You have the option to change the grouping options by any attribute, up to five attributes at a time.
 * **Filtering:** Many New Relic customers instrument [custom attributes](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom). Filtering on a specific custom attribute can be a quick way to cut through the noise of all error occurrences. 
 
+### Error profiles [#profiles]
+
+Error **Profiles** can help you answer questions like, which users were most common across all the error events during the selected time window? Or, which session was most common across all the error events during the selected time window? 
+
+You can download the attributes as CSV. You can also view the attributes breakdown when you select a row.
```

Comment:
which attributes? the error profile ones?

Reply:
that's a good question! Let me investigate.